src/ucs/sys/socket.h

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_SOCKET_H
+#define UCS_SOCKET_H
+
+#include <ucs/type/status.h>
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/ioctl.h>
+#include <net/if.h>
+#include <arpa/inet.h>
+
+
+BEGIN_C_DECLS
+
+
+/* A string to hold the IP address and port from a sockaddr */
+#define UCS_SOCKADDR_STRING_LEN  60
```

Comment:
minor: can align with other macro definitions below

Reply:
do you mean to do the following? i.e. add additional spaces between `60` and `UCS_SOCKADDR_STRING_LEN`
```
#define UCS_SOCKADDR_STRING_LEN          60

#define UCS_SOCKET_INET_ADDR(_addr)      (((struct sockaddr_in*)(_addr))->sin_addr)
#define UCS_SOCKET_INET_PORT(_addr)      (((struct sockaddr_in*)(_addr))->sin_port)

 #define UCS_SOCKET_INET6_ADDR(_addr)    (((struct sockaddr_in6*)(_addr))->sin6_addr)
#define UCS_SOCKET_INET6_PORT(_addr)     (((struct sockaddr_in6*)(_addr))->sin6_port)
```

Reply:
yes

Reply:
done